### ISSUE TITLE
Create a good README

### ISSUE THREAD

**Issue by JohnTroke** [afiuorio/ja-cards#10]
I think we need a better README to describe the program, it isn't urgent of course.

**Comment by afiuorio** [comment]
I agree. A good README should present the project, how to run, build and test and finally which license we're using.
